### PR TITLE
Add custom Javascript classes

### PR DESCRIPTION
**Pull request by tongoonamujera**
- [x] Add Html class
 - [x] Add Books class
 - [x] Add LocalStrorage class

### CODE REVIEW

**Comment by RayhanTabase** [comment]
Style Linters have been disabled since no CSS where used

**Comment by tongoonamujera** [comment]
> Style Linters have been disabled since no CSS where used

Man i think i have finished now

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # AWESOME BOOKS
 
-> In this project, I will build an application that allows users to create their own transaction lists. In this application, users can also create groups with which they can add their transactions. See live demo........
-
-## Explanation video
+> "Awesome books" is a simple website that displays a list of books and allows you to add and remove books from that list.
```
Comment (carlos-ssh) [inline README.md:4]: **Remember to keep the general requirements on mind it's mandatory on each project**
- Kindly enable your ``CSS linters`` and fix issues. after that request another code review.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @tongoonamujera , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
## **Hi @tongoonamujera,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 

Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**